tractor/_forkserver_hackzorz.py

```diff
@@ -0,0 +1,275 @@
+"""
+This is near-copy of the 3.8 stdlib's ``multiprocessing.forkserver.py``
+with some hackery to prevent any more then a single forkserver and
+semaphore tracker per ``MainProcess``.
+"""
+import os
+import socket
+import signal
+import struct
+import sys
+import errno
+import selectors
+import warnings
+
+from multiprocessing import (
+    forkserver, semaphore_tracker, spawn, process, util
+)
+from multiprocessing.forkserver import (
+        ForkServer, MAXFDS_TO_SEND, write_unsigned,
+        # _serve_one,
+)
+from multiprocessing.context import reduction
+
+
+# taken from 3.8
+SIGNED_STRUCT = struct.Struct('q')     # large enough for pid_t
+
+
+class AdultForkServer(ForkServer):
```

Comment:
Maybe `SingletonForkServer` is a better name?

Reply:
`PatchedForkServer`?